README.md

```
@@ -2,37 +2,44 @@
-      - Select the branch defined in the first step (for instance, `my-reports`)
-      - [OPTIONAL] Adjust the commit message that will appear later in the created pull request<br>(see the [What to do when reports are created](#next-step) section below).
-      - Click the green __Run workflow__ button
+1. Create a new branch (e.g. `my-reports`) from the `main` branch of your fork.
+1. Define the models by updating [`models.csv`](models.csv) (see the [Models](#models) section below for details)
+   * Once you are finished, push the new branch to your fork and verify that the check [Validate models.csv](../../actions/workflows/check-models.yml) was successful!
+1. [OPTIONAL] Adjust the OSP environment and tools by updating [`tools.csv`](tools.csv)<br>(see the [Tools](#tools) section below for details)
+   * In case of modifications: verify that the check [Validate tools.csv](../../actions/workflows/check-tools.yml) was successful!
+1. Go to the GitHub Action: [Create evaluation reports and projects](../../actions/workflows/create-evaluation_reports.yml)<br><br>
+   * Click the __Run workflow__ button 
+   * Select the branch defined in the first step (for instance, `my-reports`)
+   * [OPTIONAL] Adjust the commit message that will appear later in the created pull request<br>(see the [What to do when reports are created](#next-step) section below).
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)